packages/frontend/src/components/FlowStep/index.tsx

```diff
@@ -77,14 +77,7 @@ export default function FlowStep(
     substeps,
     shouldShowDragHandle,
     isDeletable,
-  } = useStepMetadata(
-    allApps,
-    step,
-    readOnly,
-    allowReorder,
-    isMobile,
-    isDrawerOpen,
-  )
+  } = useStepMetadata(step, true)
```

Comment:
if set to `true`, the drag handle will be shown for the for-each step. just a display issue, as its still not actually draggable

```suggestion
  } = useStepMetadata(step, allowReorder)
```